docs/src/user_guide/additional_options.md

```diff
@@ -0,0 +1,250 @@
+# Additional wflow options
+
+## [Starting the model with "warm" states](@id reinit)
```

Comment:
the state for reservoirs and lakes can cause errors easily if you forget to add/remove when switching on and off reservoirs/lakes modelling. I would move the reservoirs/lakes states in the specific reservoirs and lakes section in that same file

Reply:
Good spot, I have added a state section to the reservoir and lakes sections